domain-cc/README.md

```diff
@@ -1,5 +1,5 @@
 # Contention Classification
-Mapping contention text to actual classifications per `/disabilties` defined for [Benefits Reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data).
+Mapping contention text to actual classifications per `/classifier` defined for [Benefits Reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data).
```

Comment:
This is incorrect to change the endpoint.

Maybe rewriting this altogether is more helpful.
`/classifier` outputs classification codes outlined in `/disabilities` BRD endpoint and just include that in swagger docs instead of README??